src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/ProgrammableAndroid.java

```diff
@@ -894,7 +893,7 @@ protected void moveAndDig(Block b, BlockMenu menu, BlockFace face, Block block)
         throw new UnsupportedOperationException("Non-mining Android tried to mine!");
     }
 
-    protected boolean chopTree(Block b, BlockMenu menu, BlockFace face) {
+    protected void chopTree(Block b, BlockMenu menu, BlockFace face) {
```

Comment:
Do not change the return type on this.

Reply:
The return value is useless though, if the issue is to be solved.